ironfish/src/network/peers/connections/webSocketConnection.ts

```diff
@@ -78,11 +78,14 @@ export class WebSocketConnection extends Connection {
     }
 
     this.socket.onmessage = (event: MessageEvent) => {
+      if (!Buffer.isBuffer(event.data)) {
+        return
```

Comment:
I think in this case we should close the connection immediately, like we do below if the peer sends an invalid message. That should free up connections for valid peers, as opposed to waiting for the connection to time out.